core/common/src/main/java/alluxio/exception/runtime/AlluxioRuntimeException.java

```diff
@@ -86,6 +87,11 @@ public boolean isRetryable() {
    * @return a gRPC status exception representation of this exception
    */
   public StatusException toGrpcStatusException() {
+    Metadata trailers = getMetadata();
```

Comment:
Is this used? If `StatusException` is not used maybe it's better to not have this API?